src/Storefront/Pagelet/Header/HeaderPageletLoader.php

```diff
@@ -108,6 +108,8 @@ private function getLanguages(SalesChannelContext $context, Request $request): L
         );
 
         $criteria->addSorting(new FieldSorting('name', FieldSorting::ASCENDING));
+
+        // todo@skroblin why is this loaded here???
```

Comment:
because the search is in the header? :shrug: 

https://github.com/shopware/shopware/blob/trunk/src/Storefront/Resources/views/storefront/layout/header/search.html.twig#L2

Reply:
yey, but i moved it into the template data extension.

Reply:
then, it is probably not needed anymore here :blush: